### DISCUSSION TITLE
ao utilizar /* para abrir uma seção de comentário com um possível fim, ao tentar fechar ele duplica o asterisco

### DISCUSSION THREAD

**Discussion by Kkraties13** [dgadelha/Portugol-Webstudio#218] [Ideias e sugestões]
Ao utilizar /* para abrir uma seção de comentário com um possível fim, ao tentar fechar ele duplica o asterisco, e isso incomoda bastante e torna a função razoavelmente chata de se utilizar

**Comment by dgadelha** [comment]
Você diz sobre o `/**` ou apenas sobre o `/*`?

Se for sobre o `/**`, esse é o comportamento esperado por se tratar de um bloco de comentário, exemplo:

```portugol
/**
 * este é um bloco
 * de comentário
 */
```

Caso não seja isso, pode gravar a tela do problema para eu entender melhor?
